static/js/client.jsx

```diff
@@ -5,9 +5,29 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import DjangoCSRF from './lib/django-csrf';
 const SefariaReact = require('./ReaderApp');
+import * as Sentry from "@sentry/react";
 
 
 $(function() {
+  // Initialize Sentry, sentryDSN is defined in base.html
+  if (sentryDSN) {
+    Sentry.init({
+      dsn: sentryDSN,
+      integrations: [
+        new Sentry.BrowserTracing({
+          // Set `tracePropagationTargets` to control for which URLs distributed tracing should be enabled
+          tracePropagationTargets: ["localhost", /^https:\/\/sefaria\.org\/api/],
```

Comment:
I'm not sure what this does, but should the host here also be customiseable  for use in non prod environments?

Reply:
this looks at client side API requests and captures failures, the idea of the regex is to lower the endpoints we follow that. However I see in their docs that this feature is deprecated so I will remove the tracePropagationTargets